src/widgets/scrollbar.rs

```diff
@@ -0,0 +1,698 @@
+    }
+
+    /// Sets the orientation of the scrollbar.
+    /// Resets the symbols to [`DOUBLE_VERTICAL`] or [`DOUBLE_HORIZONTAL`] based on orientation
+    pub fn orientation(mut self, orientation: ScrollbarOrientation) -> Self {
+        self.orientation = orientation.clone();
+        self.symbol(match orientation {
+            ScrollbarOrientation::Vertical => DOUBLE_VERTICAL,
+            ScrollbarOrientation::Horizontal => DOUBLE_HORIZONTAL,
+        })
+    }
+
+    /// Sets the symbol that represents the scroller.
+    pub fn thumb_symbol(mut self, thumb_symbol: &'a str) -> Self {
+        self.thumb_symbol = thumb_symbol;
+        self
+    }
+
+    /// Sets the style that is used for the scroller.
+    pub fn thumb_style(mut self, thumb_style: Style) -> Self {
+        self.thumb_style = thumb_style;
+        self
+    }
+
+    /// Sets the symbol that represents the bar of the scrollbar.
+    pub fn track_symbol(mut self, track_symbol: &'a str) -> Self {
+        self.track_symbol = track_symbol;
+        self
+    }
+
+    /// Sets the style that is used for the bar of the scrollbar.
+    pub fn track_style(mut self, track_style: Style) -> Self {
+        self.track_style = track_style;
+        self
+    }
+
+    /// Sets the symbol that represents the beginning of the scrollbar.
+    pub fn begin_symbol(mut self, begin_symbol: Option<&'a str>) -> Self {
+        self.begin_symbol = begin_symbol;
+        self
+    }
+
+    /// Sets the style that is used for the beginning of the scrollbar.
+    pub fn begin_style(mut self, begin_style: Style) -> Self {
+        self.begin_style = begin_style;
+        self
+    }
+
+    /// Sets the symbol that represents the end of the scrollbar.
+    pub fn end_symbol(mut self, end_symbol: Option<&'a str>) -> Self {
+        self.end_symbol = end_symbol;
+        self
+    }
+
+    /// Sets the style that is used for the end of the scrollbar.
+    pub fn end_style(mut self, end_style: Style) -> Self {
+        self.end_style = end_style;
+        self
+    }
+
+    pub fn symbol(mut self, symbol: Set) -> Self {
+        self.track_symbol = symbol.track;
+        self.thumb_symbol = symbol.thumb;
+        if self.begin_symbol.is_some() {
+            self.begin_symbol = Some(symbol.begin);
+        }
+        if self.end_symbol.is_some() {
+            self.end_symbol = Some(symbol.end);
+        }
+        self
+    }
+
+    pub fn style(mut self, style: Style) -> Self {
+        self.track_style = style;
+        self.thumb_style = style;
+        self.begin_style = style;
+        self.end_style = style;
+        self
+    }
+
+    pub fn arrows(mut self, arrows: bool) -> Self {
+        if arrows {
+            self = self.begin_symbol(Some(DOUBLE_VERTICAL.begin));
+            self = self.end_symbol(Some(DOUBLE_VERTICAL.end));
+        } else {
+            self = self.begin_symbol(None);
+            self = self.end_symbol(None);
+        }
+        self
+    }
```

Comment:
I decided to remove the `arrows` method. 

So now, this will draw arrows:

```rust
            Scrollbar::default()
```

And this will not draw arrows:

```rust
            Scrollbar::default()
                .begin_symbol(None)
                .end_symbol(None)
```